docs/metrics/v3.md

```diff
@@ -343,3 +343,13 @@ The standard metrics group for GoCollector is not shown below.
 | `minio_replication_max_queued_bytes`              | `gauge` | Maximum number of bytes queued for replication since server start           | `server` |
 | `minio_replication_max_queued_count`              | `gauge` | Maximum number of objects queued for replication since server start         | `server` |
 | `minio_replication_max_data_transfer_rate`        | `gauge` | Maximum replication data transfer rate in bytes/sec seen since server start | `server` |
+
+### `/ilm`
```

Comment:
Can we add a section at the top of the document also like we have for other root level metrics like audit, notification, etc ?

Reply:
It is next to `/replication` and `/notification`

Reply:
@balamurugana I'm not asking to move it to top, but add a small section in the beginning describing what these metrics provide. e.g. refer how `/audit` is described at the beginning, and then all metrics under it are documented down the line. (just following the current style of metrics-v3 documentation - I see that this is missing in case of `/replication` too and probably needs to be fixed)

Having said that, @ravindk89 can you please review if the current style needs to be improved upon? A lot of metrics categories that were earlier intended to be under `/cluster` have now moved to their own root level endpoints, and hence the current style may need to be changed to something easier to read.

Reply:
I added like `/audit` section

Reply:
Format isn't bad - for web docs we just import the markdown file and 'promote' all the headings so that we have a clean set of H1/H2s on the entire page.

We can do that as-is with what you already have. We can make further refinements on later passes.